tests/api-testing/tests/test_alertsnew.py

```
@@ -0,0 +1,311 @@
+        "operator": ">=",
+        "frequency": 1,
+        "cron": "",
+        "threshold": 3,
+        "silence": 10,
+        "frequency_type": "minutes",
+        "timezone": "UTC",
+        "tolerance_in_secs": 0
+    },
+    "org_id": "default",
+    "destinations": [destination_alert],
+    "context_attributes": {},
```

Ruff (0.8.2)

Dictionary key literal `"context_attributes"` repeated

Remove repeated key literal `"context_attributes"`

(F601)